queries.js

```diff
@@ -0,0 +1,63 @@
+const Pool = require('pg').Pool
+const pool = new Pool({
+  user: 'admin',
+  host: 'localhost',
+  database: 'api',
+  password: 'admin_pass',
```

Comment:
If you're familiar with env variables, can you please add the db and password in a .env file

Reply:
yeah, I'll do